watertap/tools/parameter_sweep/parameter_sweep_writer.py

```diff
@@ -129,54 +138,6 @@ def _interp_nan_values(global_values, global_results):
 
         return global_results_clean
 
-    def save_results(
```

Comment:
Did this method just move locations?

Reply:
That's correct, I just moved it to the end of the file.